server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt

```diff
@@ -55,6 +55,7 @@ enum class BoardType(val id: UInt) {
 	HARITORA(18u),
 	ESP32C6DEVKITC1(19u),
 	GLOVE_IMU_SLIMEVR_DEV(20u),
+	GESTURES(21u), // https://www.gestures.space/
```

Comment:
```suggestion
	GESTURES(21u),
```
We do not ocmment on any of the types, especially with links https://github.com/SlimeVR/SlimeVR-Tracker-ESP/pull/413#discussion_r2020167835

Reply:
I thought this is what you asked about the link. Removed.